database/seed.js

```diff
@@ -0,0 +1,39 @@
+const Promise = require('bluebird');
+const faker = require('faker');
+const schema = Promise.promisifyAll(require('./data/schemaHelpers'));
+
+// note default properties: id, timestamp, and isliked
```

Comment:
Please remove the commented code blocks unless they are intended to be informative.

----------------
In addition, the Airbnb Style Guide for long comment blocks is the following pattern:
```js
/**
 * Some text here in a comment
 * even more text
*/
```

Reply:
Removed!